source/docs/software/kinematics-and-odometry/swerve-drive-kinematics.rst

```diff
@@ -236,6 +236,22 @@ By recording a set of swerve module states using :ref:`NetworkTables <docs/softw
         }
       };
 
+   .. code-block:: python
+
+      import ntcore
+      from wpimath.kinematics import SwerveModuleState
+
+      # get the default instance of NetworkTables
+      nt = ntcore.NetworkTableInstance.getDefault()
+
+      # Start publishing an array of module states with the "/SwerveStates" key
+      topic = nt.getStructArrayTopic("/SwerveStates", SwerveModuleState)
+      self.pub = topic.publish()
+
+      def periodic(self):
+        # Periodically send a set of module states
+        self.pub.set(SwerveModuleState(frontLeftState,frontRightState,backLeftState,backRightState))
```

Comment:
Oh, you're right, this should be `self.pub.set([frontLeftState,frontRightState,backLeftState,backRightState])`